### PR TITLE
chore: Update readme for GraphQL spec compability

### PR DESCRIPTION
**Pull request by calvincestari**
Trying to make it clearer for users to know which version of the GraphQL spec the client is compatible with.

### CODE REVIEW

**Comment by calvincestari** [comment]
Keeping this as a draft until we've all agreed on the messaging.

**Review by jerelmiller — APPROVED** [review]
Feel free to merge when you're ready! Thanks for adding this.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ Join 1000+ engineers at GraphQL Summit for talks, workshops, and office hours, O
 
 Apollo Client is a fully-featured caching GraphQL client with integrations for React, Angular, and more. It allows you to easily build UI components that fetch data via GraphQL.
 
+Apollo Client aims to comply with the the [Working Draft of the GraphQL specification](https://spec.graphql.org/draft/).
```
Comment (calvincestari) [inline README.md:21]:
```suggestion
Apollo Client aims to comply with the [Working Draft of the GraphQL specification](https://spec.graphql.org/draft/).
```

**Review by jerelmiller — APPROVED** [review]

**Comment by jerelmiller** [comment]
@calvincestari let me know when you're ready to merge!

**Comment by calvincestari** [comment]
> @calvincestari let me know when you're ready to merge!

@jerelmiller - it's ready to go. 🫡

**Pull request merged**